### ISSUE TITLE
Constant stream of errors in console

### ISSUE THREAD

**Issue by okfro** [noobot/SlackConnector#25]
I have a console app that's logging to Slack, and within the last update or so I have started seeing this steady stream of errors:

```
Unable to parse message:
Newtonsoft.Json.JsonReaderException: Error reading JObject from JsonReader. Path '', line 0, position 0.
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
   at SlackConnector.Connections.Sockets.Messages.Inbound.MessageInterpreter.ParseMessageType(String json)
   at SlackConnector.Connections.Sockets.Messages.Inbound.MessageInterpreter.InterpretMessage(String json)
```

I am debugging, and I have breakpoints everywhere that SlackConnector is used--but nothing is triggering. I'm set to break on all exceptions of any kind, but I'm not catching this one.

Communication to/from slack appears to be unaffected, but the noise level is now very high in the console. [This seems to be the code that logs the error](https://github.com/noobot/SlackConnector/blob/4e754ed955a172bbce21c17a24330a10681f1c80/src/SlackConnector/Connections/Sockets/Messages/Inbound/MessageInterpreter.cs)

**Comment by Workshop2** [comment]
Hey @crfroehlich thanks for reporting this.

I accepted & refactored a large pull request recently and I think I need to ensure it's stable.

Are you able to revert to the previous version if it starts causing you issues?

_Note: you can hide these errors with `SlackConnector.SlackConnector.LoggingLevel = ConsoleLoggingLevel.All;`  (that enum is supposed to be none, I will change that on the next version)_

**Comment by okfro** [comment]
@Workshop2 Hiding the errors works for me! I don't see any other issues with this release, it otherwise seems to be functioning as expected; so I will take your suggestion and hide the errors for now. Thanks!

**Comment by okfro** [comment]
Actually, on further observation, the value of `ConsoleLoggingLevel.All` seems to work as advertised :) I get a different stream of logs on every ping (no errors though). I'll try reverting to a previous version for now.

**Comment by Workshop2** [comment]
This is now resolved in 3.1.116 
https://www.nuget.org/packages/SlackConnector/3.1.116
